Cargo.toml

```diff
@@ -128,6 +124,7 @@ default = ["wat", "wast", "cache", "wasi", "engine", "emscripten", "middlewares"
 # That means: that is able to execute modules
 backend = []
 jsc = ["wasmer/jsc", "backend", "wat", "wasmer/std"]
+wasm-c-api = ["wasmer/wasm-c-api", "backend", "wat", "wasmer/std"]
```

Comment:
We should add a new one: `wamr = ["wasm-c-api"]`

Reply:
f2148f462fc398b0676a8587e97fa010b8e1e90a